portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -11,22 +11,52 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+ 
 package com.google.sps.servlets;
-
+ 
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+ 
+import com.google.gson.Gson;
+
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-
+ 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+ 
+  private List<String> names;
+ 
+  @Override
+  public void init() {
+    names = new ArrayList<>();
```

Comment:
I would need more context, but it it seems to me to be more appropriate to be initialized as const static above where it is declared... We can even take it one step further, make the conversion method static and initialize to the json string. Overall, we should avoid computation during handling time as much as possible.